storage/ctab/src/test/java/org/openscience/cdk/io/SDFWriterTest.java

```diff
@@ -370,4 +370,33 @@ public void setProgramName() {
             assertThat(mol, CoreMatchers.containsString("Bioclip"));
         }
     }
+
+    @Test
+    public void optionallyTruncateLongProperties() {
+        StringWriter sw = new StringWriter();
+        try (SDFWriter sdfw = new SDFWriter(sw)) {
+            sdfw.getSetting(MDLV2000Writer.OptWriteDefaultProperties)
+                .setSetting("false");
+            sdfw.getSetting(SDFWriter.OptTruncateLongData)
+                .setSetting("true");
+            IAtomContainer mol = TestMoleculeFactory.make123Triazole();
+            mol.setProperty("MyLongField",
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped" +
+                            "ThisIsAVeryLongFieldThatShouldBeWrapped");
+            sdfw.write(mol);
+        } catch (IOException | CDKException e) {
+            e.printStackTrace();
+        }
+        String sdf = sw.toString();
+        assertThat(sdf,
+                   CoreMatchers.containsString("ThisIsAVeryLongFieldThatShouldBeWrappedThisIsAVeryLongFieldThatShouldBeWrappedThisIsAVeryLongFieldThatShouldBeWrappedThisIsAVeryLongFieldThatShouldBeWrappedThisIsAVeryLongFieldThatShouldBeWrappedThisI\n"));
```

Comment:
If truncation is set to true, would we not expect newlines in the string?

Reply:
truncate not split, there is nothing in the spec about how to join long fields back together - only for RDfile.

Reply:
Thanks for the clarification!